File: 프로그래머스/1/77884. 약수의 개수와 덧셈/약수의 개수와 덧셈.py
def solution(left, right):
    answer = 0
    for i in range(left,right+1):
        counter = 0
        for j in range(1,i+1):
            if i % j == 0:
                counter += 1
        if counter % 2 == 0:
            answer += i
        elif counter % 2 != 0:
            answer -= i
            
    return answer

File: 프로그래머스/1/77884. 약수의 개수와 덧셈/약수의 개수와 덧셈.js
function solution(left, right) {
    var answer = 0;
    for(let i=left; i<=right; i++){
        let counter = 0;
        for(let j=1; j<=i; j++){
            if(i % j ==0){
                counter++;
            }
        }
        if(counter % 2 == 0){
            
            answer+= i;
        }
        else{
            answer-= i;
        }
    }
    return answer;
}